app/templates/components/forms/orders/order-form.hbs

```diff
@@ -278,13 +278,23 @@
         <i class="ticket icon"></i>
         {{t 'Acknowledgement'}}
       </h4>
-      <p>
+       {{#if this.isPaidOrder}}
+        <p>
         {{t
-          'By clicking "Pay Now", I acknowledge that I have read and that I agree with all the terms of services and privacy policy of Open Event.'}}
+          'By clicking "Proceed to Checkout", I acknowledge that I have read and that I agree with the '}} <a href='https://eventyay.com/terms'>terms of services</a>{{t ' and ' }}<a href='https://eventyay.com/privacy-policy'>privacy policy</a> {{ ' on this website.'}}
```

Comment:
you have created external links. create internal links. use terms and privacy-policy route.

Reply:
Okay, so It'll be solved using the <LinkTo> tag
Thanks